src/components/SignUp/SignUpFooter.tsx

```diff
@@ -56,7 +56,8 @@ const SignUpFooter: React.FC<{
     currentStep === 'privacy' ? 'Accept & Continue' : 'Continue';
 
   const nextButtonAction = async (navNext: boolean, currStep: string) => {
-    if (currStep === 'privacy') {
+    console.log('currentStep', currentStep);
```

Comment:
You'll probably want your own footer component for company signup to avoid complexity

Reply:
I was thinking this as well. Will move forward with that.